src/elements/select/multiselect.svelte

```diff
@@ -116,7 +116,7 @@ const handleEnter = () => {
 
 const handleChange = (changedOption: string) => {
   if (parsedSelected.includes(changedOption)) {
-    const newValue = [...parsedSelected.filter(item => item !== changedOption)];
+    const newValue = parsedSelected.filter(item => item !== changedOption);
```

Comment:
Most of the changes in `src` are new linting rules. For example here: filter copies arrays so ESLint removed this unnecessary spread.